lib/index.js

```diff
@@ -28,6 +28,7 @@ var internals = {
 
 internals.schema = Joi.object({
     onConnect: Joi.func(),                                  // function (ws) {}
+    onMessage: Joi.func(),                                  // function (message, reply) { reply(data); }
     onUnknownMessage: Joi.func(),                           // function (message, ws) { ws.send('string'); }
```

Comment:
Probably. I wanted to leave that out of this PR though. I'm thinking that it probably should go away since it doesn't quite work on the client side.